src/api/posts/src/data/post.js

```diff
@@ -39,7 +39,7 @@ function ensureFeed(feed) {
  * Determine the type of the post depending on the given URL.
  * The possible types are 'video' or 'blogpost'.
  *
- * @param {*} url
+ * @param {String} url
```

Comment:
Type names use lowercase, `string`, while `String` is the name of the JavaScript constructor function.